STM32F4/variants/generic_f407v/ld/common.inc

```diff
@@ -164,11 +150,43 @@ SECTIONS
          * I'm shoving these here naively; there's probably a cleaner way
          * to go about this. [mbolivar]
          */
-        _lm_heap_start = DEFINED(_lm_heap_start) ? _lm_heap_start : _end;
+        _lm_heap_start = DEFINED(_lm_heap_start) ? _lm_heap_start : __ccm_end__;
```

Comment:
Shouldn't the heap start be __ccm_start__ ?
Because the heap starts at low address and increases, while stack starts at __ccm_end__ and decreases.